Xamarin.Forms.Core/Shell/Shell.cs

```diff
@@ -538,6 +538,9 @@ ShellNavigationState GetNavigationState(ShellItem shellItem, ShellSection shellS
 
 		public static readonly BindableProperty CurrentStateProperty = CurrentStatePropertyKey.BindableProperty;
 
+		public static readonly BindableProperty FlyoutBackgroundImageSourceProperty =
+			BindableProperty.Create(nameof(FlyoutBackgroundImageSource), typeof(ImageSource), typeof(Shell), default(ImageSource), BindingMode.OneTime);
```

Comment:
Do we need the `Source` on the naming ?

Reply:
I think not.
This was done by analogy with the existing controls.
https://github.com/xamarin/Xamarin.Forms/blob/bd31e1e9fc8b2f9ad94cc99e0c7ab058174821f3/Xamarin.Forms.Core/Cells/ImageCell.cs#L7
https://github.com/xamarin/Xamarin.Forms/blob/bd31e1e9fc8b2f9ad94cc99e0c7ab058174821f3/Xamarin.Forms.Pages/ListItemControl.cs#L9
https://github.com/xamarin/Xamarin.Forms/blob/bd31e1e9fc8b2f9ad94cc99e0c7ab058174821f3/Xamarin.Forms.Pages/CardView.cs#L27
https://github.com/xamarin/Xamarin.Forms/blob/bd31e1e9fc8b2f9ad94cc99e0c7ab058174821f3/Xamarin.Forms.Pages/HeroImage.cs#L9

Reply:
those are old stuff, let's drop the source.